components/src/dynamo/vllm/omni/connectors/nixl_connector.py

```
@@ -0,0 +1,475 @@
+                    self._cancel_pending(request_id, "duplicate request id replaced")
+
+                token = str(id(readable_op))
+                deadline = time.monotonic() + max(0.0, self._pending_timeout_s)
+
+                async def _wait_release(req_id: str = request_id) -> None:
+                    try:
+                        await readable_op.wait_for_completion()
+                        logger.debug(
+                            "[DynamoOmniNixlConnector.put] RDMA complete req=%s", req_id
+                        )
+                    except Exception as exc:
```

Ruff (0.15.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)